website/docs/developer/api/api-gateway.md

```diff
@@ -213,6 +213,82 @@ The following table lists the fields in the output:
 
 The API returns status code 200 on success, 400 if `asyncOperationId` is missing, 500 if an error occurs when querying the database and 404 if not found.
 
+## Internal Reconcile report jobs API
+The `orca/datamanagement/reconciliation/internal/jobs` API call receives page index from end user and returns available internal reconciliation jobs from the Orca database.
+Internal reconcile report orphan API input invoke URL example: `https://example.execute-api.us-west-2.amazonaws.com/orca/datamanagement/reconciliation/internal/jobs`
+
+### Internal Reconcile report orphan API input
```

Comment:
should be Internal Reconcile report jobs API input

Reply:
also change `Internal reconcile report orphan API input invoke URL example` to `Internal reconcile report jobs API input invoke URL example`

Reply:
Gah. Fixed here and elsewhere.